website/content/posts/introducing-the-enterprise-contract.md

```diff
@@ -23,16 +23,16 @@ ensures the same image is being verified across all steps preventing
 tools that can do this. I’ll use [skopeo](https://github.com/containers/skopeo) since I’m familiar
 with it:
 
-```text
+{{<code bash>}}
```

Comment:
Just to check ` ```bash ` doesn't work here?

Reply:
Not without some customization of the highlightjs on load behavior. (Which is doable, but maybe for later.)